frontend/src/App.js

```diff
@@ -138,10 +138,16 @@ function App() {
                     <Typography align='left' paragraph>The IFC file is valid when it conforms to:
 
                     <ul>
-                        <li><b>Syntax</b> The STEP Physical File syntax</li>
-                        <li><b>Schema</b> The IFC schema referenced in the file including formal propositions encoded in the EXPRESS where rule and function language</li>
-                        <li><b>Rules</b> Other normative rules of the IFC specification (e.g. implementer agreements and informal propositions, and industry best practices)</li>
-                        <li><b>bSDD</b> The Validation Service validates the content of an IFC file against the requirements encoded in the bSDD domain referenced from a classification encoded in the file</li>
+                        <li><b>STEP Syntax</b> The STEP Physical File syntax</li>
+                        <li><b>IFC Schema</b> The IFC schema referenced in the file including formal propositions encoded in the EXPRESS where rule and function language</li>
+                        <li><b>Normative IFC Rules</b> Other normative rules of the IFC specification (e.g. implementer agreements and informal propositions)</li>
+                    </ul>
+
+                    <Typography align='left' paragraph>Additionally, the Validation Service performs non-normative checks including:
```

Comment:
```suggestion
                    </Typography>
                    <Typography align='left' paragraph>Additionally, the Validation Service performs non-normative checks including:
```

As mentioned by @rw-bsi